examples/app-with-rbac/src/components/App/App.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { Route, Switch } from 'react-router-dom';
+import { ROUTES } from '../../constants';
+import { Hello, Patents, ResearchDocs } from '../../pages';
+import { prefixRoute } from '../../utils/utils.routing';
+// @ts-ignore
```

Comment:
Not the best :grimacing: 
But we don't yet have a better alternative

Reply:
Is there an open issue for this? Any reason why this is not exported?

Reply:
https://github.com/grafana/identity-access-team/issues/406